test/integration/diff_test.go

```diff
@@ -18,8 +18,8 @@ import (
 var update = flag.Bool("update", false, "update the *.golden files for diff presenter")
 
 const (
-	baseURL   = "https://toolbox-data.anchore.io/grype/databases/vulnerability-db_v4_2022-07-05T08:18:22Z_39868af44fc51829a7c9.tar.gz"
-	targetURL = "https://toolbox-data.anchore.io/grype/databases/vulnerability-db_v4_2022-07-06T08:16:42Z_c840f17244dea46d0c07.tar.gz"
+	baseURL   = "https://toolbox-data.anchore.io/grype/staging-databases/vulnerability-db_v5_2022-10-14T08:22:01Z_69c99aa5917dea969f2d.tar.gz"
```

Comment:
not in scope for this PR: we need to rework this test to not be dependent on hosted DBs like this. We're thinking about deprecating beyond a time window in the future, which means this test would start to fail after 90days (or whatever window we go with). 

We could capture this as an OCI artifact, but there is also the problem that this is coupled to the current DB schema version... should we change this? or is this assumption OK? In which case, we just need a nice way to continually update the fixture when we bump the schema version.